### PR TITLE
Flat Display of Object Properties

### PR DESCRIPTION
**Pull request by nummi**
https://github.com/emberjs/ember-inspector/issues/364

screen shot 2018-10-15 at 7 40 38 pm

### CODE REVIEW

**Comment by teddyzeenny** [comment]
Love it 😍

File: ember_debug/object-inspector.js [hunk 1]
```diff
@@ -198,7 +198,7 @@ export default EmberObject.extend(PortMixin, {
     return (val instanceof EmberObject) || typeOf(val) === 'array';
   },
 
-  saveProperty(objectId, mixinIndex, prop, val) {
```
Comment (nummi) [inline ember_debug/object-inspector.js]: `mixinIndex` was not being used — confirmed with Teddy

**Comment by rwwagner90** [comment]
@nummi @teddyzeenny any idea on these tests failures?

**Comment by nummi** [comment]
No idea!

**Pull request merged**